pkg/proto/datadog/api/v1/api.proto

```diff
@@ -108,6 +108,20 @@ service AgentSecure {
         };
     };
 
+    rpc ClientGetConfigsHA(datadog.config.ClientGetConfigsRequest) returns (datadog.config.ClientGetConfigsResponse) {
+        option (google.api.http) = {
+            post: "/v1/grpc/remoteconfig/configs_ha"
+            body: "*"
+        };
+    };
+
+    rpc GetConfigStateHA(google.protobuf.Empty) returns (datadog.config.GetStateConfigResponse) {
+        option (google.api.http) = {
+            post: "/v1/grpc/remoteconfig/state_ha"
+            body: "*"
+        };
+    };
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 Suggestion:** ‏If we're adding that, let's also update https://github.com/DataDog/datadog-agent/blob/44f0a522e3f0431664c0df1bc2eb9e909729ab5f/cmd/agent/subcommands/remoteconfig/command.go so that the flare can also retrieve HAMR info